app/src/main/java/com/kickstarter/viewmodels/ResetPasswordViewModel.kt

```diff
@@ -84,14 +87,20 @@ interface ResetPasswordViewModel {
 
             resetPasswordNotification
                     .compose(errors())
-                    .map { ErrorEnvelope.fromThrowable(it) }
-                    .filter { ObjectUtils.isNotNull(it) }
+                    .map {
+                        // host -> internet related error or otherwise -> invalid email address
+                        if( it !=  null) return@map it.localizedMessage!!.contains("host")
```

Comment:
not forcing optionals, `!!` is not allowed, use the `?.let` unwrapping technique or any other technique

Reply:
I think we can simplify this line to be:
`.map { it?.localizedMessage?.contains("host") ?: false }`

If i'm not mistaken, this will return false if any of the variables are null or if the localized message does not contain "host". 

Also, nitpicky request, but could we move "host" into a constant instead of a string literal? Up to you, that is just a personal preference.